src/main.cpp

```diff
@@ -450,11 +458,20 @@ int main(int argc, char* argv[])
     mudlet::start();
 
     if (first_launch) {
```

Comment:
To ***test*** the code one will probably want to make this always `true` rather than just on the first run....!